### ISSUE TITLE
Create New Qube Dialog is not accessible using keyboard

### ISSUE THREAD

**Issue by unman** [QubesOS/qubes-issues#10988]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
4.3.1

### Brief summary
It is difficult/impossible to navigate this dialog using only the
keyboard.

### Steps to reproduce
Open "Create New Qube" dialog.
Attempt to use the keyboard.

### Expected behavior
Dialog will be usable only from keyboard.

### Actual behavior
It is somewhat difficult to tab from field to field, and there is no
indicator of the current field. Navigation seems to use a combination
of the Tab and Enter keys.
There is no indicator of which field is enabled.

Example - Input name, tab. What field is enabled? None seems to be in
force. It appears that one needs to use Input Name,Enter,Tab,Enter

Example - How does one navigate to the "Select Type" fields? Some long
sequence of Enter,Tab combinations, but what? (I have got to that field,
and found that Tab then moved between options - ordinarily one would expect
to use Up and Down, with Tab moving out of the field.)

Example - In the "Applications" field, you have to Enter+Tab to move
between elements. Why?

Example - I have no idea at what point the "Create New qube" button is
enabled, if at all.

### Additional information
Make the UX conform to normal usage:
Indicate what field/button is enabled.
Tab between fields, use UP and Down for selection in field.
Scroll the Window with Tab to Advanced Options fields.
